### PR TITLE
remove Bats submodule and work around Bats bug

### PR DESCRIPTION
**Pull request by reidpr**
Addresses #552 and #553.

### CODE REVIEW

File: test/travis.yml [hunk 1]
```diff
@@ -72,12 +71,12 @@ jobs:
       env: CH_BUILDER=buildah         TARBALL=export       INSTALL=yes
     - <<: *stage_install
       env: CH_BUILDER=buildah-runc    TARBALL=export       INSTALL=yes
+    - <<: *stage_install
+      env: CH_BUILDER=buildah-setuid  TARBALL=export       INSTALL=yes
```
Comment (j-ogas) [inline test/travis.yml]: Is this related to removing the bats submodule? If not, perhaps we should make a separate issue, which we can also close with this PR. Thoughts?

Comment (reidpr) [reply test/travis.yml]: Probably? I removed the `TARBALL=export-bats` one, which freed up a space in the stage, so I added this one.
